backend/reservations/management/commands/send_onboarding.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import sys
+from django.core.management.base import BaseCommand, CommandError
+import reservations.config as roombaht_config
+from reservations.models import Guest
+from reservations.helpers import my_url, send_email
+
+logging.basicConfig(stream=sys.stdout, level=roombaht_config.LOGLEVEL)
+logger = logging.getLogger('SendOnboarding')
+
+def onboarding_email(email, otp):
+    hostname = my_url()
+    body_text = f"""
+        BleepBloopBleep, this is the Room Service RoomBaht for Room Swaps letting you know the floors have been cleaned and you have been assigned a room. No bucket or mop needed.
+
+        After you login below you can view your current room, look at other rooms and send trade requests. This functionality is only available until Monday 11/7 at 5pm PST, so please make sure you are good with what you have or trade early.
```

Comment:
Let's change the date for this years date:
`only available until Sunday 11/5 at 5pm PST`
Still checking on confirmation for this date/time.

Reply:
confirmed and changed